spec/helpers/mime_types.rb

```diff
@@ -0,0 +1,9 @@
+def ms_mime_types
+  ['application/vnd.openxmlformats-officedocument.wordprocessingml.document; charset=binary',
+   'application/zip; charset=binary']
+end
```

Comment:
This helper's name is a bit incomplete - I would call this helper `ms_word_mime_types` :-)